tests/test_enterprise/api_client/test_lms.py

```diff
@@ -89,7 +89,8 @@ def test_enroll_user_in_course():
     course_details = {"course_id": course_id}
     mode = "audit"
     cohort = "masters"
-    expected_response = dict(user=user, course_details=course_details, mode=mode, cohort=cohort, is_active=True)
+    expected_response = dict(user=user, course_details=course_details, mode=mode, cohort=cohort, is_active=True,
```

Comment:
[style nitpick] Move args to a new line as mentioned above.

Reply:
@iloveagent57  It's done